### DISCUSSION TITLE
Linux version

### DISCUSSION THREAD

**Discussion by bellolongart** [sumatrapdfreader/sumatrapdf#3351] [Ideas]
have you think do a version multiplatform? in linux for example i haven't a pdf viewer like this

**Comment by GitHubRulesOK** [comment]
The Code can only run under Linux WinE mulator

see https://www.sumatrapdfreader.org/docs/Why-only-Windows

In Linux you have Okular

**Comment by marcosabcarvalho** [comment]
Although there are other PDF readers on Linux, and it is possible to run it through Wine, it would be very chic and elegant to have a native version for Linux. I can help develop this and even package it for Debian, Ubuntu, and equivalents.

**Reply by GitHubRulesOK** [reply]
@marcosabcarvalho 
SumatraPDF is finely customised for Windows graphics and if I run a raw MuPDF-GL on this machine it does not work without SumatraPDF windows specific mods to the base MuPDF graphics system. The WinE is where all the Linux < HAL is ported. So to convert a native WinGraphics app to native Linux X-graphics, should not be easy, especially since SumatraPDF no longer works with XP nor 32bit hence all my problems keeping a personal 32bit fork synchronised with different graphics handling just for current "Legacy!" Windows 10x32.

**Reply by kjk** [reply]
@marcosabcarvalho let's start small: can you write up detailed, step-by-step instructions on how best to run SumatraPDF on Linux under wine, to be included in docs https://www.sumatrapdfreader.org/docs/SumatraPDF-documentation

I don't use Linux so I'm not very familiar with the topic.

I believe it would be possible to make it easier to use SumatraPDF-via-wine on Linux by creating e.g. apt package, dependent on wine, so someone could do `apt-get install sumatrapdf` to install it.

I can build and host the packages but I don't have the knowledge on how to create them.
